src/app/fdctl/run/tiles/fd_gossip.c

```diff
@@ -811,10 +815,11 @@ populate_allowed_fds( fd_topo_t const *      topo,
 #define STEM_CALLBACK_CONTEXT_TYPE  fd_gossip_tile_ctx_t
 #define STEM_CALLBACK_CONTEXT_ALIGN alignof(fd_gossip_tile_ctx_t)
 
-#define STEM_CALLBACK_AFTER_CREDIT after_credit
-#define STEM_CALLBACK_BEFORE_FRAG  before_frag
-#define STEM_CALLBACK_DURING_FRAG  during_frag
-#define STEM_CALLBACK_AFTER_FRAG   after_frag
+#define STEM_CALLBACK_AFTER_CREDIT          after_credit
+#define STEM_CALLBACK_DURING_HOUSEKEEPING   during_housekeeping
```

Comment:
extra extra nit: still not quite aligned. You have 3 whitespaces here but just need one.

Reply:
I didn't like how cramped this housekeeping line looked lol. I'll fix that